app.js

```diff
@@ -1,19 +1,24 @@
-const express = require('express')
-const {getApi , getTopics, getArticleId, getArticles}  = require('./controller')
-const app = express()
+const express = require("express");
+const {
+  getApi,
+  getTopics,
+  getArticleId,
+  getArticles,
+  getComments,
+} = require("./controller");
+const app = express();
 // middleware
-app.use(express.json())
-
+app.use(express.json());
```

Comment:
Has this line of code been driven by a test? Every code written should be driven by a test or part of a test but we haven't yet got one that requires express.json() yet and therefore this code shouldn't exist at this point! :eyes: